cli/cmd/http_util.go

```diff
@@ -0,0 +1,61 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/ghodss/yaml"
+
+	"encoding/json"
```

Comment:
Can you re-order these so that the glang ones are in the top section and then "others" are after the blank line?
eg.  https://github.com/cncf/servicebroker/blob/master/k8s/service_controller/service_controller.go